app/claim/template.hbs

```diff
@@ -0,0 +1,22 @@
+{{#login-box}}
+  <div class="row">
+    <div class="col-xs-6 col-xs-offset-3">
+      <div class="panel panel-subdued panel-focused">
+        <div class="panel-heading">
+          <h3>Invitation Verification</h3>
+        </div>
+        <div class="panel-body">
+          {{#if error}}
+            <div class="alert alert-danger fade in">
+              {{error}}
+            </div>
```

Comment:
fwiw we now have `{{#bs-alert alert="danger"}}` for displaying bootstrappy alerts.